src/headers/tomcrypt_math.h

```diff
@@ -15,6 +15,12 @@
    typedef void rsa_key;
 #endif
 
+#ifndef LTC_MILLER_RABIN_REPS
+   /* Number of repetitions of the Miller-Rabin test
```

Comment:
K, I'll move this to a separate PR

Regarding "repetitions" vs. "rounds", I don't really care, I took the wording from gmp and I somehow liked it.